docs/Doxyfile

```diff
@@ -1,6 +1,7 @@
 DOXYFILE_ENCODING      = UTF-8
 PROJECT_NAME           = "FV3DYCORE"
 PROJECT_LOGO	       = image/UFSLogo.png
+PROJECT_NUMBER         = "Version 1.1.0"
```

Comment:
Don't know if this is needed.

Reply:
Technically the documentation is at the 2.x level.  This value should be set when making tagged releases.

Reply:
I have removed this in the latest commit